sql/driver/rows.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Peter Mattis (peter@cockroachlabs.com)
+
+package driver
+
+import (
+	"database/sql/driver"
+	"io"
+)
+
+type rows struct {
+	columns []string
+	// TODO(pmattis): This is unfinished. Each row needs to have len(columns)
+	// entries.
+	rows []string
+	pos  int
```

Comment:
https://golang.org/pkg/database/sql/driver/#Rows exposes an iterator interface. `pos` is the index of the iteration into `rows`. I'll add a comment to make this clearer.